tor_archivist/main.py

```diff
@@ -147,7 +107,11 @@ def main():
     bot_name = 'debug' if config.debug_mode else os.getenv('BOT_NAME', 'bot_archiver')
 
     build_bot(bot_name, __version__, full_name='u/transcribot')
-    config.archive = config.r.subreddit('ToR_Archive')
+    
+    config.archive = config.r.subreddit('tor_testing_ground')
```

Comment:
Looks like another local debugging change we need to change back. Perhaps this should be a toggle or otherwise set via some configuration option (environment variable)?

Reply:
Agreed this could probably be handled better. I like the idea of getting it from an environment variable. @itsthejoker what do you think?

Reply:
Yes, let's do an environment variable that defaults to production (`ToR_Archive`).

Reply:
Done!